runelite-client/src/main/java/net/runelite/client/plugins/motherlode/MotherlodeOreOverlay.java

```diff
@@ -71,21 +71,39 @@ public Dimension render(Graphics2D graphics)
 		int runiteFound = session.getRuniteFound();
 
 		// If no ores have even been collected, don't bother showing anything
-		// This will statement is true iff all ore counts are 0
-		if ((nuggetsFound | coalFound | goldFound | mithrilFound | adamantiteFound | runiteFound) == 0)
+		if (nuggetsFound == 0 || coalFound == 0 || goldFound == 0 || mithrilFound == 0
+			|| adamantiteFound == 0 || runiteFound == 0)
```

Comment:
This is not right.

Reply:
Whoops you're right, made a whoopsie re-typing this.